app/build.gradle

```diff
@@ -32,4 +32,10 @@ dependencies {
     implementation dependency_android.appcompat
     implementation dependency_android.design
     implementation dependency_android.constraint_layout
+
+    androidTestImplementation 'junit:junit:4.12'
```

Comment:
Can we keep putting dependencies in the dependencies.gradle file for consistency ?